core/ast/src/main/resources/org/lgna/project/ast/ArithmeticInfixExpression.properties

```diff
@@ -1,7 +1,7 @@
-INTEGER_DIVIDE = </leftOperand/> INTEGER_DIVIDE </rightOperand/>
-INTEGER_REMAINDER = </leftOperand/> INTEGER_REMAINDER </rightOperand/>
+INTEGER_DIVIDE = </leftOperand/> WHOLENUMBER_DIVIDE </rightOperand/>
```

Comment:
Please add an _ in front of NUMBER to make make it WHOLE_NUMBER_DIVIDE (and for others below)
It is more readable and consistent with the camel caps WholeNumber.
(I am not keen on either the ALL CAPS, or the length, but can't offer anything better, so adding _s is enough)

Reply:
I made the changes